lib/image.rb

```diff
@@ -1,13 +1,14 @@
 module FormatParser
-  class FileInformation
+  class Image
```

Comment:
This is one of the most valuable parts of this PR in my opinion 👍

Reply:
🙇 🙇 agreed. Also matches suuuuper well with the system I'll be using this in internally, which will let me refactor out some complexity.